src/db/ClimbSchema.ts

```diff
@@ -59,6 +59,28 @@ const MetadataSchema = new Schema<IClimbMetadata>({
   }
 }, { _id: false })
 
+const PitchSchema = new mongoose.Schema({
+  _id: {
+    type: 'object',
+    value: { type: 'Buffer' },
+    default: () => muuid.v4()
+  },
+  uuid: {
+    type: 'string',
+    default: function () { return this._id.toString() }
+  },
+  parentId: { type: String, required: true },
```

Comment:
Since `parentID` references Climb._id, the type should be `Schema.Types.Mixed`.  You can see an example in Area.climbs[] Schema.  https://github.com/OpenBeta/openbeta-graphql/blob/4ce0c36a4941e1654c24fd80aac901555489a2cf/src/db/AreaSchema.ts#L94